src/workerd/api/http.c++

```diff
@@ -1595,6 +1595,23 @@ jsg::Promise<jsg::Ref<Response>> handleHttpRedirectResponse(jsg::Lock& js,
     return js.rejectedPromise<jsg::Ref<Response>>(kj::mv(exception));
   }
 
+
+  if (FeatureFlags::get(js).getStripAuthorizationOnCrossOriginRedirect()) {
+    auto base = urlList.back().toString();
+    auto currentUrl = jsg::Url::tryParse(base.asPtr());
+    auto locationUrl = jsg::Url::tryParse(location, base.asPtr());
+    if (KJ_ASSERT_NONNULL(currentUrl).getOrigin() != KJ_ASSERT_NONNULL(locationUrl).getOrigin()) {
+          // If request’s current URL’s origin is not same origin with locationURL’s origin, then
+          // for each headerName of CORS non-wildcard request-header name, delete headerName from
+          // request’s header list.
+          // -- Fetch spec s. 4.4.13
+          // <https://fetch.spec.whatwg.org/#http-redirect-fetch>
+          //  (NB: "CORS non-wildcard request-header name" consists solely of "Authorization")
+
+          jsRequest->getHeaders(js)->delete_(jsg::ByteString(kj::str("authorization")));
+    }
```

Comment:
```suggestion
    if (KJ_ASSERT_NONNULL(currentUrl).getOrigin() != KJ_ASSERT_NONNULL(locationUrl).getOrigin()) {
      // If request’s current URL’s origin is not same origin with locationURL’s origin, then
      // for each headerName of CORS non-wildcard request-header name, delete headerName from
      // request’s header list.
      // -- Fetch spec s. 4.4.13
      // <https://fetch.spec.whatwg.org/#http-redirect-fetch>
      //  (NB: "CORS non-wildcard request-header name" consists solely of "Authorization")

      jsRequest->getHeaders(js)->delete_(jsg::ByteString(kj::str("authorization")));
    }
```

Reply:
Is this just about indenting? This is what clang-format gave me.

Reply:
Although it allowed me to fix it... I am so confused now.

Reply:
yeah, just indenting. clang-format makes me sad sometimes